stable/external-dns/Chart.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: v1
+description: Deploys external-dns into a kubernetes cluster
+name: external-dns
+version: 0.0.1
+appVersion: 0.3.0
```

Comment:
👍 I've gone for v0.4.0-alpha.2 if thats ok? I couldn't find any flags that had changed, do you have a link to any docs about that?

p.s. I'll push my changes shortly.